playbooks/publish/docs.yaml

```diff
@@ -146,4 +146,9 @@
           file:
             path: "{{ vault_token_dest }}"
             state: "absent"
+
+        - name: Remove working directory
```

Comment:
I suggest giving a name in the form "Ensure unpacked docs are removed" and adding a comment "Drop unpacked docs, since they are already uploaded to the target destination and zuul logs also contain archived copy". Without explicit explanation you will wonder few weeks later: "why?"